src/undux/GameStore.js

```diff
@@ -23,6 +23,7 @@ type State = {|
   jpDisplayStyle: JpDisplayStyleType,
   playerID: ?string,
   playerExp: number,
+  testBool: number
```

Comment:
since this is a flag, it makes more sense to store it as a bool in our store. also could you give this a more descriptive name?